eidas_node/connector/views.py

```diff
@@ -185,6 +203,16 @@ def get_light_storage(self, backend: str, options: Dict[str, Any]) -> LightStora
         """
         return import_from_module(backend)(**options)
 
+    def get_auxiliary_storage(self, backend: str, options: Dict[str, Any]) -> AuxiliaryStorage:
```

Comment:
Ok, seeing this here, I think that this should be a general utility function for both `proxy_service` and `connector`?

Or is there a reasonable assumption that someone might need to override this?

Reply:
Indeed, it should a general utility function. `backend` may be any callable so I can't think of a need to override `get_auxiliary_storage`.

Reply:
Rebased to use `get_auxiliary_storage` from the new module.